index.js

```diff
@@ -8,7 +8,8 @@ var cachedDependencies = [];
 
 var defaultOptions = {
   cache: false,
-  yes: true
+  yes: true,
+  warn: true
```

Comment:
As you wish. I don’t know what people prefer, but I haven’t seen any unhelpful warnings from Elm so far. My personal preference is to see the warnings, in order to copy top-level type declarations into my source code.